tests/installation/bootloader.pm

```diff
@@ -49,7 +50,7 @@ sub run {
         # boot
         send_key 'ctrl-x';
     }
-    compare_bootparams(\@params, [parse_bootparams_in_serial]);
+    compare_bootparams(\@params, [parse_bootparams_in_serial]) if !is_livecd;
```

Comment:
Could you state here *why* it does not make sense to compare the parameters on live media? I know that it does not work but honestly I do not know why

Reply:
Looks like different behavior of linuxrc.

Reply:
ok. Is it that the method `parse_bootparams_in_serial` is just relying on the string "command line:" which is only output by linuxrc and nothing else? how about adding a comment in the file here?